src/main/groovy/org/craftercms/cli/commands/group/AddGroupMembers.groovy

```diff
@@ -51,11 +51,11 @@ class AddGroupMembers extends AbstractCommand {
 
         def result = client.post(path, body)
         if (!result) {
-            println "Failed to add ${users} to ${groupName} group"
+            println "Failed to add '${users}' to '${groupName}'"
```

Comment:
This is not what I requested. Please don't resolve my comments, I'll do that. Please fix this.